src/compile-middleware.ts

```diff
@@ -83,7 +91,10 @@ export function babelCompile(forceCompile: boolean): RequestHandler {
       const compile = forceCompile || needCompilation(uaParser);
 
       if (compile) {
```

Comment:
Lots of unnecessary branching going on here, can you change this to a `if(!compile) { return body; }` to exit early and reduce nesting?